### ISSUE TITLE
Workflow Guidelines

### ISSUE THREAD

**Issue by NataliaLKB** [foundersandcoders/street-flowers#11]
These are just suggestions. @Neats29 and @benjaminlees do add more  to this, and please say if you disagree with any points.

* Branch names should be named off of the feature you are working on
* Commits should aim to be "safe" versions of the project. Don't be afraid to merge commits if in hindsight they really should have just been one.
* ALL Pull Requests need to be reviewed before being merged into Master. 
       * Should each PR be only reviewed by one person, or by both people? For example if I send a PR, should both of you review, or only the first person who takes a look. 
* Don't merge a PR if you think it is incomplete, or isn't tested.

Personally I would like to set up travis to run the tests just like we did for Carrier Pigeon so that we can see the tests pass or fail before merging a PR as well. What do you guys think about this?

**Comment by iteles** [comment]
We should have a way of tagging this stuff as 'style guide' so we don't lose the content into the ether!

**Comment by NataliaLKB** [comment]
@iteles agree!

**Comment by Neats29** [comment]
thanks @NataliaLKB this is great, agreed. We should further discuss tests.

**Comment by NataliaLKB** [comment]
@Neats29 I agree we need to discuss testing further. Here is a start: https://github.com/foundersandcoders/street-flowers/issues/15
